lib/client/ipam_test.go

```diff
@@ -170,8 +180,86 @@ var _ = Describe("IPAM tests", func() {
 		Entry("Assign 1 IPv4 address with AssignIP then try to release 2 IPs (release a second one)", net.ParseIP("192.168.1.1"), false, []string{"192.168.1.0/24", "fd80:24e2:f998:72d6::/120"}, net.IP{}, 0, []cnet.IP{cnet.IP{net.ParseIP("192.168.1.1")}}, nil),
 	)
 
+	DescribeTable("ClaimAffinity: claim IPNet vs actual number of blocks claimed",
+		func(inNet string, host string, cleanEnv bool, pool []string, assignIP net.IP, expClaimedIPs, expFailedIPs int, expError error) {
```

Comment:
You shouldn't need different type TestArgs for each test case within a file, but its fine to have a TestArgs per test file.

I'm not really worried about it looking weird - this test has plenty of other stuff that isn't in any other test (it uses ClaimAffinity, for example) and there is absolutely no requirement that we update the other tests every time we add new ones.